Formula/emqx.rb

```diff
@@ -0,0 +1,69 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.9.tar.gz"
+  sha256 "bc5d193a02840b1d4cb5f6c9f047003b442c5a0b4a3ccb1cfeedafacf04ff82e"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  def dashboard_version
+    "1.1.0"
+  end
+
+  def emqx_version
+    "5.0.9"
+  end
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
```

Comment:
None of these seem to be used

Reply:
They are used implicitly in the application build. Top level build command is a script, each application in the release has their own build routine.